galaxy.xsd

```diff
@@ -2099,13 +2115,12 @@ code file wraps a metadata collection to return metadata parameters wrapped
 according to the Metadata spec.
 
 ```xml
-<command>
```

Comment:
I'd restore `<command>` here and also add `<![CDATA[` (and correspondingly at the end).